dto/api_input.py

```diff
@@ -67,38 +66,53 @@ class TeamsLeagueDTO(BaseDTO):
 
 class TeamsStatisticsOrPlayersDTO(BaseDTO):
     sport_id = fields.Int(required=False, missing=None)
+    sport = fields.Str(required=False, missing=None)
     team_id = fields.Int(required=False, missing=None)
     league_id = fields.Int(required=False, missing=None)
-
+    name = fields.Str(required=False, missing=None)
+    page = fields.Int(required=False, missing=0)
+    per_page = fields.Int(required=False, missing=0)
 
 class SearchDTO(BaseDTO):
-    leagues__sport_id = fields.Int(required=False, missing=None)
-    countries__country_id = fields.Int(required=False, missing=None)
+    sport_id = fields.Int(required=False, missing=None)
+    country_id = fields.Int(required=False, missing=None)
     letter = fields.Str(required=False, missing="")
+    name = fields.Str(required=False, missing=None)
     page = fields.Int(required=False, missing=0)
     per_page = fields.Int(required=False, missing=0)
 
 
 class SportsLeagueDTO(BaseDTO):
-    leagues__sport_id = fields.Int(required=False, missing=None)
+    sport = fields.Str(required=False, missing=None)
     page = fields.Int(required=False, missing=0)
     per_page = fields.Int(required=False, missing=0)
 
 
 class GamesDTO(BaseDTO):
-    games__sport_id = fields.Int(required=False, missing=None)
-    games__league_id = fields.Int(required=False, missing=None)
-    games__country_id = fields.Int(required=False, missing=None)
+    sport_id = fields.Int(required=False, missing=None)
+    league_id = fields.Int(required=False, missing=None)
+    country_id = fields.Int(required=False, missing=None)
     status = fields.Str(required=False, missing=None)
-    games__date = fields.Date(required=False, missing=datetime.now().strftime('%Y-%m-%d'))
+    date = fields.Date(required=False, missing=datetime.now().strftime('%Y-%m-%d'))
+    date_from = fields.Date(required=False, allow_none=True)
```

Comment:
DRY principle + SOLID violation.

Will discuss on the meeting (I'm to hungry to write an essay here)